### ISSUE TITLE
Progress Bars: Indeterminate Looping Bar, Warning Type, Hardcoded to use Danger Color

### ISSUE THREAD

**Issue by ahand** [vmware/clarity#3034]
## Describe the bug

In the Dark Theme file, I set the value for the Progress Bar's warning color property to be a yellow color:

$clr-progress-warning-color: #FFD100;  //dark yellow
$clr-progress-danger-color: #D20000;  //red

However, I noticed on the Indeterminate Progress Bars page in the Kitchen Sink app that the Warning type is red, just like I'd set the Danger property to be. 

Doing some investigation, it appears that in the _progress-bars.clarity.scss file, both danger and warning types of indeterminate progress bars are hard coded to use the danger color. Instead, the warning type should be coded to use the warning property color, as separately specified. 

## How to reproduce

I made changes to the properties shown in the file:
clarity-master/src/clr-angular/utils/_theme.dark.clarity.scss

The Progress Bars file is: 
clarity-master/src/clr-angular/progress/progress-bars/_progress-bars.clarity.scss

1. Change the values in the dark theme file as noted above.
2. Run the Clarity Kitchen Sink app.
3. View the progress-bars > Indeterminate Progress Bar section of the demo.

## Expected behavior

Each special type of progress bar should use its own properties as separately specifiable.

## Versions

**App**

Kitchen Sink app as downloaded from Github on Friday, Jan. 11, 2019.

Clarity: v1.0.4

![image](https://user-images.githubusercontent.com/11655671/51134119-80f7fd00-17eb-11e9-9c2b-3cb2f1f21cbb.png)

**Comment by mathisscott** [comment]
Thanks @ahand 

Clarity generally doesn't make much use of the warning color. It looks like it was completely disregarded in this instance. I'll get this request in our queue.

**Comment by gnomeontherun** [comment]
This is the rule you'd have to add to your CSS to override this yourself. Note, this should only be an issue with looping progress bars. 

```css
.progress.loop.warning::after {
  background-color: $clr-progress-warning-color;
}
```

**Comment by mathisscott** [comment]
I verified locally that this has been fixed with the changes in #2942 

Those changes are available now in the 2.0 beta and will be released with 2.0
